.github/workflows/ci.yml

```diff
@@ -70,14 +70,35 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        width:
-          - 'w1'
-          - 'w2'
-          - 'w3'
-        height:
-          - 'h1'
-          - 'h2'
-          - 'h3'
+        device:
+          - 'w1-h1'
+          - 'w2-h1'
+          - 'w3-h1'
+          - 'w1-h2'
+          - 'w2-h2'
+          - 'w3-h2'
+          - 'w1-h3'
+          - 'w2-h3'
+          - 'w3-h3'
+        include:
+          - device: 'w1-h1'
+            test-port: 7357
+          - device: 'w2-h1'
+            test-port: 7358
+          - device: 'w3-h1'
+            test-port: 7359
+          - device: 'w1-h2'
+            test-port: 7360
+          - device: 'w2-h2'
+            test-port: 7361
+          - device: 'w3-h2'
+            test-port: 7362
+          - device: 'w1-h3'
+            test-port: 7363
+          - device: 'w2-h3'
+            test-port: 7364
+          - device: 'w3-h3'
+            test-port: 7365
```

Comment:
An ideal pseudocode might be,

```yaml
- name: Test
  run: npx percy exec -- yarn test:ember --path=dist --test-port=${{ 7357 + matrix.index }}
```

At the moment, GitHub Actions doesn't seem to provide an index out of the box. I tried using [`fromJSON()`](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson), but to no avail.